Tasks/Src/Task_Init.c

```diff
@@ -125,38 +95,26 @@ void Task_Init(void *p_arg) {
 		assertOSError(err);
 		*/
```

Comment:
Should change the function above to meet the same parameters as the others.

Reply:
It's commented out tho

Reply:
But it will be used later